message_ix/model/MESSAGE/sets_maps_def.gms

```diff
@@ -202,9 +202,13 @@ Alias(mode, mode2);
 *      - :math:`\widehat{t} \in \widehat{T}^{LAND} \subseteq \widehat{T}`
 *      - Mapping set of technology types and land use
 *    * - balance_equality (commodity,level)
-*      - :math:`c \in C^{FULL} \subseteq C`
-*      - :math:`l \in L^{FULL} \subseteq L`
-*      - Commodities and level related to `Equation COMMODITY_BALANCE_LT`_
+*      - .. math::
+*
+*           c \in C^{FULL} \subseteq C
```

Comment:
not sure why we used `C^{FULL}` here in the first place - maybe just write `c, l \in C, L`?

Reply:
Not sure either! I was just cleaning up the extra bullet item, which was breaking the list-to-table conversion.

Will make that change.